rskj-core/src/main/java/co/rsk/Start.java

```diff
@@ -194,7 +194,9 @@ private void startRPCServer() throws InterruptedException {
         web3Service = web3Factory.newInstance();
         web3Service.start();
         JsonRpcWeb3ServerHandler serverHandler = new JsonRpcWeb3ServerHandler(web3Service, rskSystemProperties.getRpcModules());
-        JsonRpcWeb3FilterHandler filterHandler = new JsonRpcWeb3FilterHandler(rskSystemProperties.corsDomains());
+        JsonRpcWeb3FilterHandler filterHandler = new JsonRpcWeb3FilterHandler(
+                rskSystemProperties.corsDomains(), rskSystemProperties.rpcAddress(), rskSystemProperties.rpcHost());
```

Comment:
How about creating an `RpcConfig` class? Maybe as a separate PR if you prefer.

Reply:
:+1: I'll add this on another PR